### DISCUSSION TITLE
ROCm GPUFORT package

### DISCUSSION THREAD

**Discussion by danielecesarini** [spack/spack#28778] [Q&A]
Hi @srekolam @arjun-raj-kuppala @haampie,

while trying to install the ROCm software platform using spack I've noticed the lack of [GPUFORT](https://github.com/ROCmSoftwarePlatform/gpufort) package. I am not sure if GPUFORT is embedded in other ROCm packages or it really lack. But in the second case, I've thought to develop a spack package for GPUFORT but, before to start, I would like to talk with you, which are the maintainers of all ROCm spack packages, because maybe you are also interested to work on this package (or you have been working on). I think that this package is fundamental to have a fully working ROCm software platform available in spack.

Thanks,
Daniele

**Comment by srekolam** [comment]
@danielecesarini   I have not worked on spack package for GPUFORT. i am trying to get some information on the GPUFORT package from my internal team. I will get back to you shortly.

**Reply by srekolam** [reply]
@danielecesarini here is what i gathered information on the GPUFORT -
GPUFORT is currently still R&D, still under heavy development and will be subject to some major refactoring in the coming weeks/months. At this point, it is not part of  an official ROCm Release. So there will be no spack package till the time it is part of Rocm release.

**Reply by danielecesarini** [reply]
I didn't know, thank you. 

Please, when you will release a stable version provide also the relative spack package.

Best,
Daniele
